example/test.py

```diff
@@ -159,23 +159,19 @@ def test_database_tables_created(self):
         count = int(psql_output.strip())
         assert_that(count, GreaterThan(0))
 
-    def get(self, path, **kwargs):
-        return requests.get(
-            'http://127.0.0.1:{}{}'.format(self.web_port, path), **kwargs)
-
-    def test_admin_site_live(self):
+    def test_admin_site_live(self, web_client):
         """
         When we get the /admin/ path, we should receive some HTML for the
         Django admin interface.
         """
-        response = self.get('/admin/')
+        response = web_client('/admin/')
```

Comment:
I'd prefer to have an object that we can call `.get()` or whatever on, but it's not really worth more than a couple of extra lines of setup. Maybe `requests` can give us something like that?